test/verifiedTests/testSBML/testWriteSBML.m

```diff
@@ -36,15 +45,7 @@
 else
     warning(['The file', fullFileNamePath, ' does not exist and could not be deleted.']);
 end
-
-load('Ec_iJR904.mat')
-model = convertOldStyleModel(model);
-% use a different deck to convert a COBRA model to an SBML file
-modelSBML = convertCobraToSBML(model);
```

Comment:
I would alter that function to just call writeSBML (which it kind of does). The inputs of that function were ignored anyways (and it was just kept for backward compatability). Even the old writeSBML was unable to create any SBML Level 2 models and I would strongly discourage giving that option. 
I altered the function accordingly.